index.js

```diff
@@ -49,9 +56,14 @@ function Hermes (opts) {
     connectionUrl.push(':');
     connectionUrl.push(opts.port);
   }
-  connectionUrl = connectionUrl.join('');
+  connectionUrl = [
```

Comment:
I personally don't like reusing vars (especially when it's changes type. In this case it was Array and became String)